client/app/components/guildroster/guildroster.service.ts

```diff
@@ -0,0 +1,24 @@
+import {Injectable} from 'angular2/core';
```

Comment:
This file should be in /client/app/services and conform in structure to the existing ones.